apps/condo/domains/acquiring/schema/AcquiringIntegrationContext.js

```diff
@@ -55,6 +57,31 @@ const AcquiringIntegrationContext = new GQLListSchema('AcquiringIntegrationConte
             },
         },
 
+        reason: {
+            schemaDoc: 'Contract number and date. Basis for invoicing',
+            type: Text,
+            isRequired: false,
+        },
+
+        email: {
+            schemaDoc: 'Acquiring payments report will be sent to this email',
+            type: Text,
+            kmigratorOptions: { null: true, unique: false },
+            hooks: {
+                resolveInput: ({ resolvedData }) => {
+                    if (isUndefined(resolvedData.email)) return undefined
+                    if (isNull(resolvedData.email) || isEmpty(resolvedData.email)) return null
+                    return normalizeEmail(resolvedData['email']) || resolvedData['email']
+                },
+            },
+        },
+
+        recipient: {
+            ...RECIPIENT_FIELD,
+            schemaDoc: 'Requisites from contract. Are used for invoicing in case of transit accounts or missing requisites',
+            isRequired: false,
+        },
```

Comment:
Usually in case of transit bank you have multiple recipients, how are we going to handle this case? 

Also, just a take to think on: maybe this data should lie inside `rb` service? As some kind of mapper, which sees recipient, parse that this is transit bank recipient and then maps it to some other place?

Reply:
Changed documentation for the field. Recipient for context will be used in case of dealing with receipts connected to not confirmed recipients. Transit account is only a special case.